### PR TITLE
fixed PHPDoc

### PR DESCRIPTION
**Pull request by OskarStark**
I am targetting this branch, because this is BC.

## Subject

Fixed `@author` tag an removed useclass descriptions

### CODE REVIEW

File: Admin/AbstractAdmin.php [hunk 1]
```diff
@@ -48,7 +48,7 @@
 use Symfony\Component\Validator\ValidatorInterface as LegacyValidatorInterface;
 
 /**
- * @author  Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
```
Comment (OskarStark) [inline Admin/AbstractAdmin.php]: I wonder, why this isn't fixed by PHP-CS-Fixer 😕 

//cc @greg0ire @Soullivaneuh

Comment (greg0ire) [inline Admin/AbstractAdmin.php]: No idea… do you know which fixer is supposed to fix this?

Comment (OskarStark) [inline Admin/AbstractAdmin.php]: > No idea… do you know which fixer is supposed to fix this?

no

File: Form/DataTransformer/LegacyModelsToArrayTransformer.php [hunk 2]
```diff
@@ -17,11 +17,9 @@
 use Symfony\Component\Form\Exception\UnexpectedTypeException;
 
 /**
- * Class ModelsToArrayTransformer.
+ * This class should be used with Symfony <2.7 only and will be deprecated with 3.0.
```
Comment (OskarStark) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: @greg0ire IMO a `NEXT_MAJOR` is missing here, right?

Comment (greg0ire) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: Yes, absolutely.

Comment (OskarStark) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: and a deprecation, too, right?

Comment (greg0ire) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: Not sure. If people are stuck with 2.3 and have no alternative to using this class, they should not get a deprecation notice.

Comment (greg0ire) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: So maybe a conditional deprecation.

Comment (OskarStark) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: Can I overhand this to you, you are more familar with this 😄  and anyway, this is put of scope of this PR

Comment (greg0ire) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: Ok, I'll do it.

**Comment by OskarStark** [comment]
please merge @greg0ire

**Comment by greg0ire** [comment]
Thanks @OskarStark

File: Form/DataTransformer/LegacyModelsToArrayTransformer.php (hunk 2 above)
Comment (greg0ire) [inline Form/DataTransformer/LegacyModelsToArrayTransformer.php]: See #4264

**Pull request merged**